### PR TITLE
add Invezo case study

### PR DESCRIPTION
**Pull request by argshook**
This PR adds new pages to questdb.io/case-study/invezo

### CODE REVIEW

**Comment by amyshwang** [comment]
1. I have provided three icons with wording to be added.
2. The preview image has the usual problem of being squashed. Would this image be a better fit:
image I cropped it from https://www.invezo.com/.

File: src/pages/case-study/invezo.tsx [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+import caCss from "../../css/case-study/card.module.css"
+import juCss from "../../css/case-study/jumbotron.module.css"
+import seCss from "../../css/section.module.css"
+import ouCss from "../../css/case-study/outcome.module.css"
+import { logos } from "../../assets/logos"
+
+const values = [
+  {
+    description: "Use case: market data",
```
Comment (amyshwang) [inline src/pages/case-study/invezo.tsx:17]:
```suggestion
    description: "Use case: Market data",
```

**Review by amyshwang — COMMENTED** [review]
market data to Market data.

File: src/pages/case-study/invezo.tsx [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+          </p>
+
+          <p className="font-size--large">
+            Here is a comparison of the same query between MongoDB for time
+            series and QuestDB:
+          </p>
+
+          <CodeBlock title="MongoDB Query">{mongoDBQuery}</CodeBlock>
+
+          <p className="font-size--large">Query execution time: 5.4 seconds</p>
```
Comment (tris0laris) [inline src/pages/case-study/invezo.tsx:246]: MongoDB query execution time

File: src/pages/case-study/invezo.tsx [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+
+          <CodeBlock title="QuestDB Query">{questDBQuery}</CodeBlock>
+
+          <p className="font-size--large">Query execution time: 0.1 seconds</p>
```
Comment (tris0laris) [inline src/pages/case-study/invezo.tsx:250]: QuestDB query execution time

Comment (argshook) [inline src/pages/case-study/invezo.tsx:250]:
```suggestion
          <p className="font-size--large">QuestDB query execution time: 0.1 seconds</p>
```

**Review by amyshwang — COMMENTED** [review]
Adding @pswu11's Slab comment to PR.

File: src/pages/case-study/invezo.tsx [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+            different situation for crypto assets, which is a brand new asset
+            class. There is a lack of tools to analyze market price behavior and
+            underlying valuations. Invezo pulls on-chain and off-chain metrics
+            to understand the behavior of crypto assets to a much deeper level,
+            while offering superior charting capabilities and also providing an
+            API.
+          </p>
+
+          <p className="font-size--large">
+            Emmett Miller, co-founder of Invezo, tells us about his journey
```
Comment (tris0laris) [inline src/pages/case-study/invezo.tsx:167]:
```suggestion
            Emmett Miller, co-founder and CEO, Invezo, tells us about his journey
```

**Review by tris0laris — APPROVED** [review]

**Pull request merged**